### PR TITLE
feat: made an increment on slidedeck

### PR DESCRIPTION
**Pull request by EmilMunksoe**

### CODE REVIEW

File: slidedeck/slides.md [hunk 1]
```diff
@@ -22,35 +17,270 @@ transition: slide-left
...
+
+At the end you sync changes to a git server with `push` in order to save work and synchronize changes with your colleagues or automation systems.
+
+```bash
+
+git add <file> # Add a file to "staging"
+
+git commit -m <message> # Records all changes that are staged by add with documentation on the work done in the files
+
+git push -u origin <branch> # Push all missing commits from localhost to a git server syncing your records with a server for collaboration
```
Comment (rne-mft) [inline slidedeck/slides.md:46]:
```suggestion
git push -u origin <branch> # Push all missing commits from localhost (your machine) to a git server syncing your records with a server for collaboration
```

File: slidedeck/slides.md [hunk 2]
```diff
@@ -22,35 +17,270 @@ transition: slide-left
...
+layout: two-cols
+---
+
+# Basics of Git: Client-server
+
+Git works as a client server paradigm 
+<br>
+<br>
+
+Git works by a client (a user) downloads the source code from a `repository`, makes edit locally and syncs these edits to `origin` (the server of which you downloaded the source code). <br><br>
```
Comment (rne-mft) [inline slidedeck/slides.md:61]:
```suggestion
Git works by a client (a user) cloning the source code from a `repository`, performing edits locally and pushing these edits to `origin` (the server from which you downloaded the source code). <br><br>
```

File: slidedeck/slides.md [hunk 3]
```diff
@@ -22,35 +17,270 @@ transition: slide-left
...
+cat README.md
+git magic
+```
+
+NOTE: There is a cheatsheet at the bottom of the README.md in the root of the repository.
+
+---
+
+
+# What is Slidev? asdasd test
```
Comment (rne-mft) [inline slidedeck/slides.md:283]:
```suggestion
```

File: slidedeck/slides.md [hunk 4]
```diff
@@ -22,35 +17,270 @@ transition: slide-left
...
 
 Slidev is a slides maker and presenter designed for developers, consist of the following features
```
Comment (rne-mft) [inline slidedeck/slides.md:285]:
```suggestion
```

**Review by rne-mft — CHANGES_REQUESTED** [review]
I think it looks fine. Is it possible to see the slides already from the PR version before it gets merged? If so, that would be nice, it's easier for me to grasp the flow if I can run through them.

File: slidedeck/slides.md (hunk 3 above)
Comment (EmilMunksoe) [reply slidedeck/slides.md:283]: Keep for now to make inspiration when you develop slides

**Comment by EmilMunksoe** [comment]
@rne-mft See the README.md in the directory slidedeck. There are instructions on how to spin up the slidedeck locally.

**Review by rne-mft — APPROVED** [review]

**Pull request merged**